src/docker/compose.local.yaml

```diff
@@ -1,63 +1,48 @@
-# yamllint disable rule:empty-lines rule:key-ordering rule:brackets
+# yamllint disable rule:empty-lines rule:key-ordering
 # cspell:word localnet
+
+# Services in this file are EXCLUSIVELY for running a local testnet.
 ---
+include:
+# This setup allows us to only have to specify only one compose file.
+- path:
+  - 'compose.yaml'
+  - 'processor.overrides.yaml'
+name: 'emojicoin-dot-fun'
 services:
-  aptos-node:
+  localnet:
     network_mode: 'host'
-    image: 'econialabs/aptos-cli:4.1.0'
-    profiles: ["test-harness"]
-    ports:
-    - '50051:50051'
-    - '8070:8070'
-    - '8090:8090'
-    - '8080:8080'
-    - '8081:8081'
+    container_name: 'localnet'
+    build:
+      context: '../../'
+      dockerfile: 'src/docker/localnet/Dockerfile'
     healthcheck:
-      test: ['CMD', 'curl', 'http://localhost:8070/']
+      test:
+      - 'CMD'
+      - 'sh/healthcheck.sh'
       interval: '5s'
       timeout: '5s'
-      retries: 10
-      start_period: '60s'
-    # In order to start the node with the indexer API; i.e., run
-    # `aptos node run-localnet --with-indexer-api`, the Docker socket must
-    # be mounted to the container. This mount enables the container to start a
-    # new container. Note that this is *not* 'Docker-in-Docker', but rather
-    # 'Docker-outside-Docker'. That is, the container can act as the host in
-    # terms of creating and orchestrating other containers but still appears as
-    # a sibling to the existing containers and any new containers it creates.
+      retries: 5
+      start_period: '30s'
+    entrypoint:
+    - '/bin/bash'
+    - '/app/sh/run-localnet.sh'
```

Comment:
I'd remove this as well.

Since this is a core part of the container and how it functions, it only makes sense to put it there IMO.

Also, if we were to change it, we only need to change it in one place.